web/src/pages/MemoDetail.tsx

```diff
@@ -156,12 +165,15 @@ const MemoDetail = () => {
                       }}
                     >
                       {VISIBILITY_SELECTOR_ITEMS.map((item) => (
-                        <Option key={item} value={item} className="whitespace-nowrap">
+                        <Option key={item} value={item} className="whitespace-nowrap" disabled={disableOption(item)}>
                           {t(`memo.visibility.${item.toLowerCase() as Lowercase<typeof item>}`)}
                         </Option>
                       ))}
                     </Select>
-                  </>
+                    <div className="flex-grow">
```

Comment:
And this warning text should be removing too.

Reply:
Done @boojack